fix: discrepancies in 0.13.0 fee docs (#1078)
starknet-io/starknet-docs: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/fee-mechanism.adoc

* stem:[$\ell$] is the number of contract deployments.
* stem:[$0.9$] is a 10% cost reduction that considers that the sequencer does not incur additional costs for repeated updates to the same storage slot within a single block.
* stem:[$w$] is the xref:#calculation_of_computation_costs[`CairoResourceFeeWeights`] vector.
* stem:[$\text{calldata_cost}$] is 612 gas per 32-byte word (~100 gas is charged for on-chain hashing the happens for every sent word).

> * stem:[$\text{storage_write_cost}$] is the 20,000 gas, the cost associated with allocating a new storage cell on Ethereum.

Reviewer: ```suggestion
* stem:[$\text{storage_write_cost}$] is 20,000 gas, the cost of allocating a new storage cell on Ethereum.
```
